integration_tests/dbt_project.yml

```diff
@@ -66,7 +65,7 @@ seeds:
         APPLIES_TO_ID: "{{ 'INT64' if target.type == 'bigquery' else 'bigint' }}"
         ENDS_AT: "timestamp"
         STARTS_AT: "timestamp"
-    order_data:
+    orders_data:
```

Comment:
Same comment from the source to get your thoughts on if we should still test for the `order` name in our integration tests

Reply:
Agreed--added!